src/backend/TrafficCourts/Citizen.Service/Mappings/MappingProfile.cs

```diff
@@ -0,0 +1,21 @@
+﻿using AutoMapper;
+using TrafficCourts.Citizen.Service.Models.Dispute;
+using TrafficCourts.Messaging.MessageContracts;
+
+namespace TrafficCourts.Citizen.Service.Mappings
+{
+    public class NoticeOfDisputeToMessageContractMappingProfile : Profile
+    {
+        public NoticeOfDisputeToMessageContractMappingProfile()
```

Comment:
There are no unit tests for this mapping profile.

Reply:
Added unit test for `NoticeOfDispute` to `SubmitNoticeOfDispute` message contract mapping.